### PR TITLE
remove version banner from 8.0

### PR DESCRIPTION
**Pull request by akeller**
## What is the purpose of the change

OOTB Docusaurus puts up a banner stating older docs versions are "unmaintained," which is not entirely true. 

This PR removes the banner from 8.0 since OOTB, we only have 3 options - no banner, "unmaintained," or "unreleased" (as in `next`).  

Shoutout to @remcowesterhoud for starting this discussion 😄! 

## Are there related marketing activities

No.

## When should this change go live?

This should not be merged until feedback is collected internally.

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
This will require specifying exactly what older versions are still maintained (removing the banner).

This doesn't give a great visual cue to the user that they are on an older version. We would need to explore something custom.

https://user-images.githubusercontent.com/1051363/229162555-e5e179ac-75a2-4d97-8e21-da09a328c813.mov

**Comment by akeller** [comment]
@pepopowitz I added you as a reviewer for the implementation, but also for the ongoing overhead. Every new versioned release we'll need to make an adjustment to include the previous 18 month versions.

We'll also need to do something similar with Optimize, but I'd need to check with the team how many versions or months they support.

**Review by pepopowitz — COMMENTED** [review]
Re: ongoing maintenance -- we should at least add content to https://github.com/camunda/camunda-platform-docs/blob/main/howtos/release-procedure.md#perform-a-minor-release describing the need to also update this configuration when doing a release. If we wanted to get fancy about it, we could update [the versioning script](https://github.com/camunda/camunda-platform-docs/blob/main/hacks/cutNewVersions.sh#L4-L8)  to make the change for us. 

@akeller -- would you mind adding a bit to that versioning guide about what needs to be done & why, and I'll look into adding it to the script as I prepare for the upcoming release?

File: howtos/release-procedure.md [hunk 1]
```diff
@@ -65,7 +65,18 @@ To create the new versions:
...
+5. Ensure the "unmaintained" banner does not appear for supported versions. We currently support all versions of Camunda Platform 8 since none are older than 18 months.
+
+```javascript
+// 👋 When cutting a new version, remove the banner for maintained versions by adding an entry. Remove the entry to versions >18 months old.
+   versions: {
+      "8.0": {
+         banner: "none",
+      },
+   },
+```
```
Comment (pepopowitz) [inline howtos/release-procedure.md:77]: ❤️ ❤️ ❤️

**Review by pepopowitz — APPROVED** [review]
💪 Thank you!!!!!

**Comment by akeller** [comment]
I was going to keep this in draft, but based on all the feedback I've received internally, no one can definitively tell me if this is confusing customers or not.

I'd like to move forward with removing the banner and then plan to remove the banner for 8.1 when 8.2 goes lives. This will clearly show 8.0, 8.1, and 8.2 are all supported and maintained (as intended).

**Pull request merged**